backend/src/data/migrations/20210304183602-create-users-table.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize
+  .query('CREATE EXTENSION IF NOT EXISTS pgcrypto;')
+  .then(() => queryInterface.sequelize.transaction(transaction => Promise.all([
+    queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal('gen_random_uuid()')
+      },
+      name: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      surname: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      birthdate: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      sex: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      type: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
```

Comment:
You should use enum types here. See here: https://github.com/BinaryStudioAcademy/bsa-2021-healthcare/pull/2/files?file-filters%5B%5D=.js#diff-ec3dfa7ba63d0684bbf43b6de2680eec8ed498e0acf8bda39475818f2b619cfb
And then declared them in the model https://github.com/BinaryStudioAcademy/bsa-2021-healthcare/pull/2/files#diff-ec3dfa7ba63d0684bbf43b6de2680eec8ed498e0acf8bda39475818f2b619cfb